packages/ui/src/views/repo/repo-summary/components/summary-panel.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Badge,
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+  IconProps,
+  Spacer,
+  Text
+} from '@/components'
+
+interface DetailsProps {
+  id: string
+  iconName: 'tube-sign' | 'open-pr' | 'tag' | 'branch' | IconProps['name']
+  name: string
+  count: number
+}
```

Comment:
These don't appear to be props for a `Details` component, but rather items in an array, maybe `DetailsItem` would be a more appropriate name? Or just `Detail`?
```suggestion
interface DetailsItem {
  id: string
  iconName: 'tube-sign' | 'open-pr' | 'tag' | 'branch' | IconProps['name']
  name: string
  count: number
}
```